ethwrite/test/adapter.test.ts

```diff
@@ -0,0 +1,141 @@
+import { assert } from 'chai'
+import { Requester } from '@chainlink/external-adapter'
+import { assertSuccess, assertError, startChain } from '@chainlink/adapter-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../src/adapter'
+import { deploy } from '../deploy_contract'
+
+// using DELAYED ROOT SUITE in order to start the chain and deploy the contract
+setTimeout(async function () {
```

Comment:
Not sure about this approach. Could a `before` work?

Reply:
Spent some time to make it working with `before`, but async initialization does not work on `describe`/`context`(https://github.com/mochajs/mocha/issues/2975).

Async calls could be resolved inside `it` though.

Reply:
Opened a low priority issue #321 just so we remember this